src/model/Patient.ts

```diff
@@ -18,4 +19,5 @@ export default interface Patient extends AbstractDBModel, Name, ContactInformati
   allergies?: Allergy[]
   diagnoses?: Diagnosis[]
   notes?: Note[]
+  labs?: Lab[]
```

Comment:
I don't think that labs should be a property of a Patient at the database level. A Patient doesn't really "own" a lab a patient is just a property of the lab, which is why I think it makes sense to "link" the patient to the lab in the Lab entity. This will also help with security models going forward since someone who might not be able to see patient information can see lab information.

@fox1t thoughts here?

Reply:
you are right,
then I propose remove this because the link already exists since the `Lab.ts` contains already the patient id at the database level.

and in Patient Lab Tab I would use LabRepository to find all the labs related to this patient, and then render them in the Lab Tab.

does that sound good?

Reply:
sounds good to me!

Reply:
I have done, the discussed changes,
please let me know if it is ok

Reply:
@rubencgt code changes look good! there is just a failing test to take care of. I think there may be a missing mock somewhere. See the failing check for the logs.